tests/NuGet.Services.AzureSearch.Tests/SearchService/SearchTextBuilderFacts.cs

```diff
@@ -11,13 +11,15 @@ public class SearchTextBuilderFacts
     {
         public class V2Search : FactsBase
         {
+            private const string MaxId = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
```

Comment:
This might be clearer with something like:

```suggestion
            private const string MaxId = new string('a', 123);
```

I don't feel strongly about this, feel free to keep it as is if you prefer it

Reply:
I chose this so that I can use `[InlineData(...)]` which requires a const.

Reply:
I left a comment saying the length.